kubetest/e2e.go

```diff
@@ -393,6 +436,8 @@ type deployer interface {
 	IsUp() error
 	SetupKubecfg() error
 	Down() error
+	FederationUp() error
```

Comment:
A federation control plane with its own API Server and Controller manager deployed on one of the clusters. That cluster, called the host cluster, can be deployed using any of the available deployment mechanisms.